ghostjs-core/src/ghostjs.js

```diff
@@ -84,7 +84,7 @@ class Ghost {
       this.setDriverOpts({parameters: ['-jsconsole']})
     } else if (this.testRunner.match(/chrome/)) {
       const program = spawn(ChromeGhostDriver.path, [], {
-        cwd: root,
+        cwd: process.cwd(),
```

Comment:
I was getting errors running this with the value `root` and saw you made a similar change recently @KevinGrandon so I did it here too.